R/convenience-functions.R

```diff
@@ -209,3 +209,68 @@ log_shift <- function(x,
   }
   log(x + offset, base = base)
 }
+
+
+
+
+#' @title Set unit of a single forecast manually
+#'
+#' @description Helper function to set the unit of a single forecast manually.
+#' This simple function keeps the columns specified in `forecast_unit` (plus
+#' additional protected columns, e.g. for true values, predictions or quantile
+#' levels) and removes duplicate rows.
+#'
+#' @inheritParams score
+#' @param forecast_unit character vector with the names of the columns that
+#' uniquely identify a single forecast
+#' @return A data.table with only those columns kept that are relevant to
+#' scoring or denote the unit of a single forecast as specified by the user.
+#'
+#' @importFrom data.table ':=' is.data.table copy
+#' @author Nikos Bosse \email{nikosbosse@@gmail.com}
+#' @export
+#' @references Transformation of forecasts for evaluating predictive
```

Comment:
why is this reference here?

Reply:
?

Reply:
Aaah those conversations were hidden!
You can't ever link enough to the paper! :D 
Probably copypasting from somewhere.. I removed it :)